css-brochure-form/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>LEARN MORE</h1>
+    <p>
+      COMPANY EMAIL<br />
+      <input
+        type="text"
+        placeholder="jake@google.com"
+        name="username"
+        size="50"
+        maxlength="30" /><br />
+      NAME<br />
+      <input
+        type="text"
+        placeholder="jake smith"
+        name="username"
+        size="50"
+        maxlength="30" /><br />
+      WHERE ARE YOU HIRING<br />
```

Comment:
You should not use `<br />`.  It's super bad practice. If you are using this to try and get the `input` on a new line, think about the `display` property and what value we can give it to take up a new line